src/stylesheets/components/_components.overlay.scss

```diff
@@ -0,0 +1,50 @@
+.c-menu {
```

Comment:
Update the file name from _components.overlay.scss to _components.menu.scss so it matches the component

Reply:
I think he did this in another branch but have not merge properly :p